### PR TITLE
chore: adding testnet2 toml file

### PR DESCRIPTION
**Pull request by JonRay15**
Currently only includes API details for the datanode we have deployed. 

As you add your own SSL enabled datanodes you can raise PRs including config for these.

Similarly when we are ready to deploy Console we will add config for this.

### CODE REVIEW

File: testnet2/testnet2.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  [API.REST]
+    Hosts = ["https://api.validators-testnet.vega.xyz"]
+
+[Console]
+  LocalPort = 1847
+  URL = ""
+
+[TokenDApp]
+  LocalPort = 1848
+  URL = "https://validator-testnet.governance.vega.xyz/"
```
Comment (gordsport) [inline testnet2/testnet2.toml:25]:
```suggestion
  URL = "https://validator-testnet.governance.vega.xyz/"

```

**Review by gordsport — COMMENTED** [review]
Add new line to end of file

**Comment by qmin80** [comment]
Need an approve?

**Review by qmin80 — APPROVED** [review]
All endpoint is working.

**Review by MM0819 — APPROVED** [review]

**Pull request merged**